zaza/charm_lifecycle/utils.py

```diff
@@ -24,6 +24,14 @@
 BUNDLE_DIR = "./tests/bundles/"
 DEFAULT_TEST_CONFIG = "./tests/tests.yaml"
 DEFAULT_MODEL_ALIAS = "default_alias"
+DEFAULT_DEPLOY_NAME = 'default{}'
+
+ModelDeploy = collections.namedtuple(
+    'ModelDeploy', ['model_alias', 'model_name', 'bundle'])
+EnvironmentDeploy = collections.namedtuple(
+    'EnvironmentDeploy', ['name', 'model_deploys', 'run_in_series'])
```

Comment:
I like the use of ``namedtuple`` for this, and I really like the prospect of being able to toggle serial vs. parallel runs at some point!